Python/Default.sublime-keymap

```diff
@@ -1,25 +1,110 @@
 [
-    // Auto-pair quotes even after string modifiers.
-    // Copied over from the default bindings with modifications to `preceding_text`
-    // and an added selector condition.
+    // Suppress default auto-pairing bindings to avoid unwanted interference
+    { "keys": ["\""], "command": "insert", "args": {"characters": "\""}, "context":
+        [
+            { "key": "setting.auto_match_enabled" },
+            { "key": "selector", "operand": "source.python - string.quoted.double" },
```

Comment:
I am slightly uncertain about `- string.quoted` because it disables auto-pairing double quotes in triple-quoted documentation comments or multi-line strings.

Reply:
Multiline strings may require special treatment to support auto-pairing of quotes within, but this may be part of another PR.